### ISSUE TITLE
IllegalStateException in PhpStorm

### ISSUE THREAD

**Issue by jdreesen** [gtache/intellij-lsp#60]
Hi, I just tried to get this plugin running using this guide: https://getpsalm.org/docs/language_server/#phpstorm

But somehow the PhpStorm Event Log only shows an exception :(

PhpStorm 2018.3.4
Build #PS-183.5429.47, built on February 8, 2019
JRE: 1.8.0_152-release-1343-b26 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0

```
java.lang.IllegalStateException: Duplicate RPC method textDocument/semanticHighlighting.
	at org.eclipse.lsp4j.jsonrpc.services.ServiceEndpoints.lambda$getSupportedMethods$0(ServiceEndpoints.java:113)
	at org.eclipse.lsp4j.jsonrpc.services.AnnotationUtil.findRpcMethods(AnnotationUtil.java:66)
	at org.eclipse.lsp4j.jsonrpc.services.ServiceEndpoints.getSupportedMethods(ServiceEndpoints.java:90)
	at org.eclipse.lsp4j.jsonrpc.services.ServiceEndpoints.getSupportedMethods(ServiceEndpoints.java:82)
	at org.eclipse.lsp4j.jsonrpc.Launcher$Builder.getSupportedMethods(Launcher.java:370)
	at org.eclipse.lsp4j.jsonrpc.Launcher$Builder.createJsonHandler(Launcher.java:381)
	at org.eclipse.lsp4j.jsonrpc.Launcher$Builder.create(Launcher.java:305)
	at org.eclipse.lsp4j.launch.LSPLauncher.createClientLauncher(LSPLauncher.java:106)
	at com.github.gtache.lsp.client.languageserver.wrapper.LanguageServerWrapperImpl.start(LanguageServerWrapperImpl.scala:291)
	at com.github.gtache.lsp.client.languageserver.wrapper.LanguageServerWrapperImpl.connect(LanguageServerWrapperImpl.scala:169)
	at com.github.gtache.lsp.PluginMain$.$anonfun$editorOpened$2(PluginMain.scala:135)
	at com.github.gtache.lsp.PluginMain$.$anonfun$editorOpened$2$adapted(PluginMain.scala:132)
	at scala.Option.foreach(Option.scala:257)
	at com.github.gtache.lsp.PluginMain$.$anonfun$editorOpened$1(PluginMain.scala:132)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:314)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by gtache** [comment]
Sorry about that, it's fixed in 1.5.2, it will be available via auto-update as soon as JetBrains approves it (or immediately if you install it from the github release)

**Comment by jdreesen** [comment]
Thx for the quick fix! :)
